src/app/containers/ChartbeatAnalytics/canonical/index.jsx

```diff
@@ -1,26 +1,29 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import Helmet from 'react-helmet';
 import { canonicalChartbeatPropTypes } from '../../../models/propTypes/chartbeatAnalytics';
 
 const chartbeatSource = '//static.chartbeat.com/js/chartbeat.js';
 
 const CanonicalChartbeatBeacon = ({ chartbeatConfig }) => {
-  const chartbeatConfigRef = useRef(chartbeatConfig);
+  const [firstLoadConfig] = useState(chartbeatConfig);
+
+  const hasMounted = useRef(false);
 
   useEffect(() => {
-    if (chartbeatConfigRef.current !== chartbeatConfig) {
+    if (hasMounted.current) {
       // eslint-disable-next-line no-unused-expressions
       window.pSUPERFLY && window.pSUPERFLY.virtualPage(chartbeatConfig);
```

Comment:
can we move this `window.pSUPERFLY` into the conditional and then remove the eslint disable line?

Reply:
Should now be resolved @jroebu14